### ISSUE TITLE
2019-7-17

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#59]
101.Read an article about logistic regression in python; done; do we need to separate cost and grad? Yes; separate cost and gradient; change cost function to computeCost; done; (2019-6-15)
102.What is initial theta? Done; change name of copy function; done; check our cost function with copied function; change Sigmoid to lower case; done; check out gradient with with copied ones; done; (2019-6-15)
103.Python 分号; delete; done; (2019-6-15)
104.Plot boundary; give up; learn how to set jingdu; use round; done; check whether two gradient is same; yes, let us change them to hypothesis; done; (2019-6-16)
105.Go through ex2_reg; (2019-6-16)
106.Deal with mapFeature; hold; (2019-6-16)
107.Let us see the result of neural network of octave; done; how to load mat with python; we still need neural network; see backprop algorithm; (2019-6-16)
108.Gradient descent => NN = > RNN
109.那么深层网络和浅层网络相比有什么优势呢？简单来说深层网络能够表达力更强。
